tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/deid/fitbit_dateshift_test.py

```diff
@@ -118,21 +118,35 @@ def setUp(self):
           (802, date(2016, 05, 10), datetime(2016, 05, 10, 12, 45, 00), timestamp('2016-05-10 12:45:00 UTC'), "jane doe")
         """)
 
+        s_query = self.jinja_env.from_string("""
+        INSERT INTO `{{fq_table_name}}`
+        (person_id, p_date, p_datetime, p_timestamp, p_name)
+        VALUES
+          -- setting day to the 11th to make it easier to calculate the shifted date --
```

Comment:
I think this comment is accurate for `a_query` but not `s_query`.  Could it be modified?